controllers/nonadmin_controller_test.go

```diff
@@ -296,6 +313,68 @@ func TestEnsureRequiredSpecs(t *testing.T) {
 	if deployment.Spec.Template.Spec.Containers[0].Image != defaultNonAdminImage {
 		t.Errorf("Deployment has wrong Image: %v", deployment.Spec.Template.Spec.Containers[0].Image)
 	}
+	if len(deployment.Spec.Template.Annotations[enforcedBackupSpecKey]) == 0 {
+		t.Errorf("Deployment does not have Annotation")
+	}
+	previousEnforcedBackupSpec := deployment.DeepCopy().Spec.Template.Annotations[enforcedBackupSpecKey]
```

Comment:
lets use a different variable name here for `previousEnforcedBackupSpec`, something more clearer - maybe `previousEnforcedBackupSpecAnnotation` or just `previousAnnotationValue`
This also shadows declaration of 'previousEnforcedBackupSpec' in nonadmin_controller. go

Reply:
done